downstream/modules/aap-hardening/con-external-credential-vault.adoc

```diff
@@ -11,4 +11,4 @@ Secrets management is an essential component of maintaining a secure automation
 
 * Ensure that there are no unauthorized users with access to the system, and ensure that only users who require access are granted it. {ControllerNameStart} encrypts sensitive information such as passwords and API tokens, but also stores the key to decryption. Authorized users potentially have access to everything.
 
-* Use an external system to manage secrets. In cases where credentials need to be updated, an external system can retrieve updated credentials with less complexity than an internal system. External systems for managing secrets include CyberArk, HashiCorp Vault, Microsoft Azure Key Management, and others. For more information, see the link:https://docs.ansible.com/automation-controller/4.4/html/userguide/credential_plugins.html#secret-management-system[Secret Management System] section of the {ControllerNameStart} User Guide v4.4.
\ No newline at end of file
+* Use an external system to manage secrets. In cases where credentials need to be updated, an external system can retrieve updated credentials with less complexity than an internal system. External systems for managing secrets include CyberArk, HashiCorp Vault, {Azure} Key Management, and others. For more information, see the link:https://docs.ansible.com/automation-controller/4.4/html/userguide/credential_plugins.html#secret-management-system[Secret Management System] section of the {ControllerNameStart} User Guide v4.4.
```

Comment:
For the Automation Controller UG you can use {ControllerUG}
Admin guide is {ControllerAG}